### PR TITLE
First task

### PR DESCRIPTION
**Pull request by p-nedelchev**
First task from Exceptions. Throws an exception if the string is not numeric.

### CODE REVIEW

File: src/com/clouway/first_task/Sumator.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+public class Sumator {
+
+
+    /**
+     * Parses the content of rwo numeric strings and calculates their sum
+     * @param a String first numeric string
+     * @param b String second numeric string
+     * @return Integer the sum of a and b
+     */
+    public Double sum(String a, String b) {
```
Comment (EmilGeorgiev) [inline src/com/clouway/first_task/Sumator.java]: What will happen if we pass ```null``` as parameter. What exception will rise?

Comment (p-nedelchev) [inline src/com/clouway/first_task/Sumator.java]: I added NullPointerException in the catch block.

**Comment by EmilGeorgiev** [comment]
Folder  .idea/  should be in .gitignore

**Comment by EmilGeorgiev** [comment]
LGTM

**Pull request merged**